src/plugins/intel_gpu/src/kernel_selector/kernels/swiglu/swiglu_kernel_base.h

```diff
@@ -6,18 +6,18 @@
 
 #include "kernel_base_opencl.h"
 #include "kernel_selector_params.h"
-#include "intel_gpu/op/swiglu.hpp"
+#include "ov_ops/swiglu.hpp"
```

Comment:
TO FIX: This header is not visible here
```
intel_gpu/src/kernel_selector/kernels/swiglu/swiglu_kernel_base.h:9:10: 
fatal error: ov_ops/swiglu.hpp: No such file or directory
```

Reply:
Link this module with openvino::core::dev?

Reply:
Linked `openvino::runtime::dev` also added TRANSFORMATION_API macro to export `shape_infer` function, it will be refactored as a part of 157615.